src/backend/ai-reviewer-api/pom.xml

```diff
@@ -246,20 +243,19 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
-				<version>2.22.2</version>
 			</plugin>
 
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-failsafe-plugin</artifactId>
-				<version>2.22.2</version>
 			</plugin>
 
 			<!--openapi server autogen-->
 			<plugin>
 				<groupId>org.openapitools</groupId>
 				<artifactId>openapi-generator-maven-plugin</artifactId>
-				<version>5.1.1</version>
+				<!-- Anything >= 6.3.0 is broken. Spring seems to scan for components twice which breaks at runtime when trying to create the application context. -->
+				<version>6.2.1</version>
```

Comment:
This one was tricky. 7.0.0 is the latest version, however for some reason nothing > 6.2.1 seems to work for this project as the spring context cannot load - a component scans seems to happen twice (but only for this project, not for others, like mail-it). This is fortunate since 6.1.0 is the version <useJakartaEe> was added to support Java 17.